modules/Core/includes/endpoints/RegisterEndpoint.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+/**
+ * @param string $username The username of the new user to create
+ * @param string $uuid (optional) The Minecraft UUID of the new user
+ * @param string $email The email of the new user
+ * 
+ * @return string JSON Array
+ */
+class RegisterEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'register';
+        $this->_module = 'Core';
+        $this->_description = 'Register a new user, and send email verification if needed.';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.